site/pages/gc-form-en.hbs

```diff
@@ -0,0 +1,197 @@
+						<li class="checkbox">
+							<input type="checkbox" id="checkbox_4">
+							<label for="checkbox_4">Option 4</label>
+						</li>
+					</ul>
+					<hr>
+					<h6>Small checkboxes</h6>
+					<ul class="list-unstyled gc-form">
+						<li class="checkbox checkbox-small">
+							<input type="checkbox" id="confirmation-checkbox">
+							<label for="confirmation-checkbox">Single option</label>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0 text-success"><span class="glyphicon glyphicon-ok-circle"></span> Correct use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Use large checkboxes when one or more selections can be made to answer a question. It can also be used for a checklist, with short labels that do not break across lines.</li>
+				<li>Use small checkboxes when making a checklist with items that would break across multiple lines or when there is a single item that can be checked.</li>
+			</ul>
+			<h5 class="mrgn-tp-0 text-danger"><span class="glyphicon glyphicon-remove-circle"></span> Incorrect use</h5>
+			<p>Compliance point(s):</p>
+			<ul>
+				<li>Do not use if there is a need for users to enter a detailed answer</li>
+				<li>Do not use the small checkbox pattern in wizards or questions in forms</li>
+			</ul>
+		</div>
+		<div class="col-md-4">
+			<h5 class="mrgn-tp-0">Code</h5>
+			<pre class="prettyprint prettyprinted" style=""><code><span class="com">// Large checkboxes:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_1&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_1&#x22;&#x3E;Option 1&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_2&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_2&#x22;&#x3E;Option 2&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_3&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_3&#x22;&#x3E;Option 3&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+	&#x3C;li class=&#x22;checkbox&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;checkbox_4&#x22;&#x3E;
+		&#x3C;label for=&#x22;checkbox_4&#x22;&#x3E;Option 4&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;
+
+<span class="com">// Small checkboxes:</span>&#x3C;ul class=&#x22;list-unstyled gc-form&#x22;&#x3E;
+	&#x3C;li class=&#x22;checkbox checkbox-small&#x22;&#x3E;
+		&#x3C;input type=&#x22;checkbox&#x22; id=&#x22;confirmation-checkbox&#x22;&#x3E;
+		&#x3C;label for=&#x22;confirmation-checkbox&#x22;&#x3E;Single option&#x3C;/label&#x3E;
+	&#x3C;/li&#x3E;
+&#x3C;/ul&#x3E;</code></pre>
+		</div>
+	</div>
+	<h3>Radio buttons</h3>
+	<div class="row">
+		<div class="col-md-4">
+			<div class="panel panel-default">
+				<div class="panel-body">
+					<h5 class="mrgn-tp-0">Appearance</h5>
```

Comment:
The ```h4``` heading is missing here. 

You might want to put the heading inside the ```.panel-heading``` section instead of being inside the ```.panel-body```. That way you wont need to set the margin top to 0.  Like in this example: https://wet-boew.github.io/wet-boew-styleguide/design/panels-en.html#default

Reply:
Fixed heading hierarchy and put appearance in panel heading